src/lib/containers/FullWidthAlert.tsx

```diff
@@ -157,7 +156,7 @@ function FullWidthAlert(props: FullWidthAlertProps) {
         />
         <ButtonFromConfig
           config={primaryButtonConfig}
-          variant="secondary"
+          variant="primary"
```

Comment:
Primary button should have primary style.
https://sagebionetworks.jira.com/browse/SWC-6158

Reply:
The style referred to in the design is 'sds-primary'

```suggestion
          variant="sds-primary"
```

Could be worth opening a ticket to replace all old button styles with these new ones